ghost/admin/app/templates/mentions.hbs

```diff
@@ -0,0 +1,10 @@
+<section class="gh-canvas">
+    <GhCanvasHeader class="gh-canvas-header">
+        <h2 class="gh-canvas-title" data-test-screen-title>
+            Mentions
+        </h2>
+    </GhCanvasHeader>
+    {{#each this.mentionsList as |post|}}
```

Comment:
would `as |mention|` make more sense? Or is there a reason we use post?

Reply:
Absolutely. Changed it to mention earlier and my commit after. :)

Reply:
Sorry, did I comment on an outdated commit? 🤦‍♂️